mzLib/Omics/Fragmentation/TerminusSpecificProductTypes.cs

```diff
@@ -1,8 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Proteomics.Fragmentation
+﻿namespace Omics.Fragmentation
 {
     public class TerminusSpecificProductTypes
```

Comment:
I believe this class belongs in Proteomics. Transcriptomics will return a different fragmentation terminus from identical product types